app/src/test/mocks/mockRemote.ts

```diff
@@ -57,7 +57,7 @@ export function resetFetcher() {
 }
 
 export function mockFetcher(promise: Promise<Response>, reportingPromise?: Promise<Response>) {
-    fetcher.fetcher = new ReportingFetcher();
+    fetcher.fetcher = new AdminFetcher()
```

Comment:
I've deleted the `ReportingFetcher` altogether, but admin still uses the old fetcher, so we can't get rid of this quite yet.